pages/Auto-Disenchanter.md

```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
+It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
+You place the slimefun or vanilla tool in the left slot along with a book, and a indicator will appear showing the progress and how much energy it has. Upon completion, the right side will produce the respective tool, unenchanted, and a enchanted book holding the enchants of that tool.
```

Comment:
an indicator*
showing the progress on how much*
the respective tool, disenchanted* (technically unenchanted is correct but I feel this is more accurate)
an enchanted book*